### DISCUSSION TITLE
Doc Suggestion: Performance + ky

### DISCUSSION THREAD

**Discussion by jack828** [invertase/react-native-firebase#5695] [General]
Hey. Thanks for the great library!!

I just completed a refactor of my app's API request mechanism to use https://github.com/sindresorhus/ky as it provides me with a nice abstraction over fetch.

I notice the documentation has a tutorial [integrating with axios](https://rnfirebase.io/perf/axios-integration) I wondered if the community/maintainers would be happy with me submitting a PR for a guide adding in a page for `ky`.

Sorry if this is in the wrong section - not quite sure where it belongs!

**Comment by mikehardy** [comment]
That would be fantastic, I'm certain the ky package isn't going anywhere (that is: it will be useful for a long time), so any help for others on using it well would be great!
